aten/src/ATen/native/Normalization.cpp

```diff
@@ -455,8 +457,12 @@ std::tuple<Tensor, Tensor, Tensor, Tensor, int64_t> _batch_norm_impl_index(
 
     // don't return view of input, don't return empty tensor because it will break gradient chain
     auto out = input.clone();
-    if (weight.defined()) out = out * weight[0];
-    if (bias.defined()) out = out + bias[0];
+    if (weight.defined() && weight.numel() > 0) {
```

Comment:
This will need a comment for what the semantics are when `weight` and/or `bias` have no elements